Documentation/install/vmware/vmware-terraform.md

```diff
@@ -154,9 +153,9 @@ $ terraform plan \
 $ terraform apply \
   ../../platforms/vmware
 ```
-Shortly after running `terraform apply` new worker machines will appear on Tectonic console. This change may take several minutes.
+After running `terraform apply` new worker machines will appear in Tectonic Console. This change may take several minutes.
 
-#### Scaling master nodes
+### Scaling master nodes
```

Comment:
for bonus points, replace "master" with "controller" throughout -- but not required for this alpha platform

Reply:
we should pick a standard for this, and apply throughout the product and doc set all at once, and get Rob and Reza on board.

Reply:
I think we already did -- or at least Rob and I did long ago. But in principle, I agree.